mlir/tools/mlir-tblgen/RewriterGen.cpp

```diff
@@ -1172,9 +1172,22 @@ void PatternEmitter::emitRewriteLogic() {
       os << val << ";\n";
   }
 
+  auto processSupplementalPatterns = [&]() {
+    int numSupplementalPatterns = pattern.getNumSupplementalPatterns();
+    for (int i = 0, offset = -numSupplementalPatterns;
+         i < numSupplementalPatterns; ++i) {
+      DagNode resultTree = pattern.getSupplementalPattern(i);
+      auto val = handleResultPattern(resultTree, offset++, 0);
+      if (resultTree.isNativeCodeCall() &&
```

Comment:
I think this is using the wrong one as base, wasn't https://reviews.llvm.org/D157032 the base for this?

Reply:
Yes, this is based on that. That patch got reverted and then relanded as https://reviews.llvm.org/rGd22965f0d623392fdf87e5a09a0276aa70e8dfed.

Reply:
Sorry I had gotten confused with one section from another review.

Reply:
No worries! Thanks for the confirmation!